### PR TITLE
chore: fix typos and grammar in playbook

### PR DESCRIPTION
**Pull request by Racer159**
This PR addresses some typographical and grammatical errors in the playbook documentation available at https://engineering.rise8.us/practice-playbook/ that I noticed perusing here on GitHub.  Let me know if any additional changes are needed or if there is a different path / process to follow.

I did notice some other _potential_ consistency and word choice / flow issues, but decided to not change those at this time. (such as references to both `main` and `master` in different sections of the document).

Given the license on this repo, I also wasn't sure if there was a Contributor Licensing Agreement but accept and agree to the following:

- [x] - I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.  I grant Rise8, and those who receive the Submission directly or indirectly from Rise8, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third parties.

(^ mostly derived from the Microsoft CLA sample: https://opensource.microsoft.com/pdf/microsoft-contribution-license-agreement.pdf - feel free to use or discard these Submissions in any way you see fit)

### CODE REVIEW

**Comment by jmiller-rise8** [comment]
@Racer159 Appreciate your contribution. We just need to review with the team to align on the license.

**Review by jwills-r8 — APPROVED** [review]
Thank you for the fixups ✨

**Comment by jmiller-rise8** [comment]
@Racer159 We've updated our license and contributing docs. Are you still okay making the contribution under the new license?

**Review by jmiller-rise8 — APPROVED** [review]

**Review by jmiller-rise8 — CHANGES_REQUESTED** [review]
@Racer159 We've updated our license and contributing docs. Are you still okay making the contribution under the new license. Please resolve this conversation when ready.

**Review by jmiller-rise8 — COMMENTED** [review]
@Racer159 We've updated our license and contributing docs. Are you still okay making the contribution under the new license. Please resolve this conversation when ready.

**Review by jmiller-rise8 — APPROVED** [review]

File: docs/practice-playbook.md
Comment (jmiller-rise8) [inline docs/practice-playbook.md:1]: @Racer159 We've updated our license and contributing docs. Are you still okay making the contribution under the new license. Please resolve this conversation when ready.

Comment (Racer159) [reply docs/practice-playbook.md:1]: Yep I'm good with that license

**Pull request merged**